src/algorithms/tracking/OrthogonalTrackSeedingConfig.h

```diff
@@ -78,5 +78,13 @@ namespace eicrecon {
     float  m_seedConfMaxZOrigin_forw = 150.0 * Acts::UnitConstants::mm;
     float  m_minImpactSeedConf_forw = 1.0 * Acts::UnitConstants::mm;
 
+    //////////////////////////////////////
+    ///Seed Covariance Error Matrix
+    float m_Loc_a_Error = 1.5 * Acts::UnitConstants::mm;    //Error on Loc a
+    float m_Loc_b_Error = 1.5 * Acts::UnitConstants::mm;    //Error on Loc b
```

Comment:
Comment here and throught out the use of this parameter is to use lower case for consistency in the naming scheme (start lower case, capital for new words):
```suggestion
    float m_loc_a_Error = 1.5 * Acts::UnitConstants::mm;    //Error on Loc a
    float m_loc_b_Error = 1.5 * Acts::UnitConstants::mm;    //Error on Loc b
```